allure-csharp.sln

```diff
@@ -15,19 +15,21 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Allure.SpecFlowPlugin", "Al
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Allure.SpecFlowPlugin.Tests", "Allure.SpecFlowPlugin.Tests\Allure.SpecFlowPlugin.Tests.csproj", "{69C462C3-E955-4B0C-9F1F-2ED5AE1BF270}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Allure.Features", "Allure.Features\Allure.Features.csproj", "{9F2B97FF-6798-4A69-9520-CE3491396F3F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Allure.Features", "Allure.Features\Allure.Features.csproj", "{9F2B97FF-6798-4A69-9520-CE3491396F3F}"
```

Comment:
Looks like accidental solution file changes.

Reply:
Rolled back all the solution changes and removed new project.